charts/kubescape-cloud-operator/templates/operator/role.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.operator.enabled }}
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Values.operator.name }}
+  namespace: {{ .Values.ksNamespace }}
+rules:
```

Comment:
The Operator needs the ability to create and delete cronjobs, configmaps, and secrets in the Kubescape namespace

Reply:
added